streams/src/test/java/org/apache/kafka/test/KStreamTestDriver.java

```diff
@@ -225,20 +225,9 @@ public ProcessorNode processor(String name) {
     }
 
     public void flushState() {
-        final ProcessorNode current = currNode;
-        try {
-            for (StateStore stateStore : context.allStateStores().values()) {
-                final ProcessorNode processorNode = topology.storeToProcessorNodeMap().get(stateStore);
-                if (processorNode != null) {
-                    currNode = processorNode;
-                }
-                stateStore.flush();
-            }
-        } finally {
-            currNode = current;
-
+        for (StateStore stateStore : context.allStateStores().values()) {
```

Comment:
This is a leftover work that was not introduced in this PR: before we use the `topology.storeToProcessorNodeMap()` to remember the linked processor node of the state store, so we can set it while flushing and possibly forwarding to children processor nodes. When we add KIP-63 we store that information in `DirtyEntry` so we do not need this any more, but forgot to remove it.

This cleanup is mainly for removing those not-needed-anymore maps from topology builder